aviary/interface/utils/check_phase_info.py

```diff
@@ -227,7 +227,7 @@ def check_phase_info(phase_info, mission_method):
         return
     else:
         raise ValueError(
-            "Invalid mission_method. Please choose either 'FLOPS', 'GASP', 'simple', or 'solved'.")
+            "Invalid mission_method. Please choose from 'HEIGHT_ENERGY', 'TWO_DEGREES_OF_FREEDOM', 'SIMPLE', or 'SOLVED'.")
```

Comment:
so we don't have to keep updating enums as they change, could we do something like:
```
possible_values = ["'"+e.value+"'" for e in Equations_of_Motion]
possible_values[-1] = "or " + possible_values[-1]
raise ValueError("Invalid mission_method. Please choose from " + ", ".join(possible_values) + ".")
```

Reply:
Thanks Carl, fun idea, added it and a test for it.